index.html

```diff
@@ -56,6 +56,7 @@ <h1>Welcome to MPI-SHH Summer School</h1>
 											<li>Application open: June 1st, 2021.</li>
 											<li>Application deadline: June 25th, 2021.</li>
 											<li>Registration fee: free!</li>
+											<li>Participants who attend classes and submit a poster will receive 8 ETCS and a certificate of participation.</li>

```

Comment:
Just one change of words ("present" instead of "submit"):
Participants who attend classes and **present** a poster will receive 8 ETCS and a certificate of participation.